Gemfile

```diff
@@ -53,6 +53,10 @@ group :development, :test do
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
 end
 
+group :production do
+  gem 'pg'
```

Comment:
`gem 'pg'`に修正し、pg gem インストールしました！
Rosseta用に`arch -arm64`が必要だったようです。

```
arch -arm64 gem install pg
Building native extensions. This could take a while...
Successfully installed pg-1.4.6
Parsing documentation for pg-1.4.6
Installing ri documentation for pg-1.4.6
Done installing documentation for pg after 3 seconds
1 gem installed
```